Dockerfile

```diff
@@ -1,6 +1,8 @@
 # Dockerfile
 FROM almalinux:9
 
+ARG PYTHON=${PYTHON}
```

Comment:
Does this argument get supplied from the CI builds of Docker images?

And what about from `tools/build-docker`?

Reply:
Looks like you grabbed an old commit, it now has a default.

Reply:
fyi, the PR is still in draft, so nothing is stable